src/client.ts

```diff
@@ -298,29 +298,65 @@ interface CurrentPhaseResponse {
 export interface Phase {
   effective?: Date
   features: Features[]
+  trial?: boolean
 }
 
+export interface CancelPhase {}
+
 const isDate = (d: any): d is Date =>
   d && typeof d === 'object' && d instanceof Date
 
 export const isPhase = (p: any): p is Phase =>
   p &&
   typeof p === 'object' &&
   (p.effective === undefined || isDate(p.effective)) &&
+  (p.trial === undefined || typeof p.trial === 'boolean') &&
```

Comment:
use types?

Reply:
This is a type checking function, "using types" is what's going on here.  TypeScript compiles to untyped JS, you gotta roll your own isX methods (which it makes easier by "helpfully" complaining until they're correct).

Reply:
One can't `x is Phase`? In Go it's like `_, ok := x.(Phase)` where ok is false if the underlying type of x is not Phase. TS would have built-in support for that.